_posts/2016-05-03-scaling-cto-talk.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout:       post
+title:        Hot To Scale As A CTO - Why Course Correction Is Inefficient (Video)
+author:       on_freund
+summary:
+image:        //res.cloudinary.com/wework/image/upload/c_lfill,f_auto,fl_progressive,g_north,h_1400,w_1400/v1461959537/engineering/scaling_cto_talk.jpg
+categories:   engineering process culture
+wistia_video: wistia_async_sbkrpcark4
+---
+
+### The Scaling CTO Talk at the Wheelhouse CTO Summit at the NASDAQ
+
+As VP Engineering at WeWork, I had a great time attending the [Wheelhouse CTO Summit held at the NASDAQ](http://summits.wheelhouse.io/newyork/index.html) a few months ago.
```

Comment:
I see your point now about the "VP of Engineering" here, I feel like it's self evident by the "CTO Summit" title and you have it in your badge too, so probably not necessary.